styx-common/src/main/java/com/spotify/styx/util/ReplayEvents.java

```diff
@@ -35,7 +35,6 @@ private ReplayEvents() {
     throw new UnsupportedOperationException();
   }
 
-  // TODO: fix NPath complexity
```

Comment:
Codacy doesn't seem to think that the NPath complexity issue has been resolved

Reply:
Weird. This method is not that complex now I think.

Reply:
Now it has been resolved.